README.md

```diff
@@ -116,4 +116,29 @@ This will be the replacement for NoneLab. It will be built on top of defcon.
 ### Other stuff.
 
 - We need to look through the various modules in RoboFab and see if there are any that we should retain. For example, the classic gString.
-- We need to consider how to handle the naming of environment specific methods. An environment may have a `font.blahBlahBlah(foo, bar)` method. In 10 years, we may want to implement our own version of `font.blahBlahBlah()` and we may not want the same API as the environment's API.
\ No newline at end of file
+- We need to consider how to handle the naming of environment specific methods. An environment may have a `font.blahBlahBlah(foo, bar)` method. In 10 years, we may want to implement our own version of `font.blahBlahBlah()` and we may not want the same API as the environment's API.
+
+## Testing
+
+Tests can be run using the [`py.test`](http://pytest.org/latest/) library. To set up the test suite to run locally, run the following commands:
+
+```bash
+cd ~/path/to/fontParts
+pip install -e .
+py.test
+```
+
+You can also run all tests against various versions of Python using [`tox`](https://tox.readthedocs.org/en/latest/). You will need to have several Python's installed: `python2.6`, `python2.7`, `python3.4`, and `pypy`. Consider using [`pyenv`](https://github.com/yyuu/pyenv) to install each version. For example (check what the most recent versions to install with `pyenv install --list`)
+
+```bash
+pyenv install 2.6.9
```

Comment:
are we really going to support python2.6?
note that we dropped this in fonttools because it was too pain to keep supporting 2.6 with a single py23 codebase.